cpp/src/barretenberg/dsl/acir_format/pedersen.hpp

```diff
@@ -7,6 +7,8 @@ namespace acir_format {
 // P = xG + bH
 struct PedersenConstraint {
     std::vector<uint32_t> scalars;
+    uint32_t hash_index;
```

Comment:
can this be moved to the bottom of the struct and be defaulted so the change isn't breaking?

Reply:
I'm not sure how this would work when `read` is called; I think you would need to do a conditional read, but imagine that this is in the middle of the buffer, then its not easy to know if the next byte is for the next constraint or if its the hash_index

Reply:
Yes, I think there is no way of making this a non-breaking change with our current serialization